### PR TITLE
Enh/read the doc

### PR DESCRIPTION
**Pull request by Slowblitz**
This is the new documentation for read the doc

### CODE REVIEW

**Comment by SylvainTakerkart** [comment]
> Hi @SylvainTakerkart can you check the content of introduction in the ` docs/source/ando.rst` file it needs to be updated, and also can you give us to @JuliaSprenger or me the admin right that we can do the configuration on read the doc pls .

I've upgraded both of you from Maintainer to Administrator of the AnDO project... I'm now gonna check the content of the ando.rst file ;)

**Comment by SylvainTakerkart** [comment]
Just pushed an update of index.rst ... I completily changed its content... I'm not sure whether it's worth adding other things... We could also wonder whether it's really worth having a readthedoc site... WDYT?

**Comment by SylvainTakerkart** [comment]
We should probably discuss before moving further...

**Comment by JuliaSprenger** [comment]
I think it would still be good to have version controlled documentation of the AnDO package out there, as we discussed a couple of weeks ago. This way we can keep the documentation / specifications of AnDO linked to specific versions of the code and still have all of them accessible via the different doc versions on readthedocs.
Also having a reduced more user friendly version of the specifications as developed on docs.google could help not scaring potential users by the complexity of the complete documentation.

**Comment by Slowblitz** [comment]
Also It's maybe a good idea to have it , for the INCF talk .

**Comment by SylvainTakerkart** [comment]
ok, fair points ;)

so, what we need, besides the intro is:
- a list of the tools with a very short description of what they do (checker, generator, nnw2bids converter under dev)
- a simplified version of the specs (importantly: with a snapshot view of a dataset, for example the reach2grasp dataset, similarly to what is on the BIDS homepage, with the right part only: https://bids.neuroimaging.io/assets/img/dicom-reorganization-transparent-white_1000x477.png)

anything else? WDYT?

File: docs/source/conf.py
Comment (JuliaSprenger) [inline docs/source/conf.py:23]:
```suggestion
author = 'Jeremy Garcia, Julia Sprenger, Sylvain Takerkart'
```

File: docs/source/index.rst
Comment (JuliaSprenger) [inline docs/source/index.rst:17]: @Slowblitz I think you forgot to add some text here... the last sentence is incomplete.

**Comment by SylvainTakerkart** [comment]
Hi @Slowblitz ! The image you just uploaded contains several errors with respect to the specs:
- the exp-XXX directory is not required anymore (it's the top level directory = what you show as "dataset")... 
- there are underscores in the session name, they are forbidden!
- maybe others...

Also, I would put real names from a real dataset! Can you take a look at https://gin.g-node.org/sprenger/multielectrode_grasp/src/bep_animalephys? this is our example dataset! (but some files at the root directory are forbidden there... just keep the sub-XX directories!). To make sure your example on the image is ok, you can create it on disk and run the AnDOChecker on it ;)

File: docs/source/index.rst
Comment (SylvainTakerkart) [reply docs/source/index.rst:17]: I think it's my text, ahah ;)

File: docs/source/index.rst
Comment (JuliaSprenger) [inline docs/source/index.rst:25]: the current example image still has invalid underscores in the session folder label. It should read `ses-20200102` without the trailing `_001_land` as the session label is now allowed to contain non-alphanumeric characters like underscores.
Also the file `exp-name` file is not part of AnDO.

**Comment by JuliaSprenger** [comment]
@Slowblitz can you add the link to the latest compilation of the docs here, so it is easier to compare the code changes to the output?

File: docs/source/index.rst
Comment (Slowblitz) [reply docs/source/index.rst:17]: @JuliaSprenger This is @SylvainTakerkart 's commit so ...

**Comment by Slowblitz** [comment]
Here's the link https://ando.readthedocs.io/en/enh-read_the_doc/

File: docs/source/index.rst
Comment (SylvainTakerkart) [reply docs/source/index.rst:17]: > @JuliaSprenger This is @SylvainTakerkart 's commit so ...

yep, but @Slowblitz , if you see that there's something like this, you should let me know about it ;)

**Comment by SylvainTakerkart** [comment]
Hi @Slowblitz ! For the image: could you put all the files at the top of the image, and the two directories sub-i ad sub-l at the bottom? that will make the image more readable I think!

Other than this, it looks good! I'll just make some small final adjustments to the text later (or more probably tuesday morning)...

Thanks ;)

**Comment by Slowblitz** [comment]
So I manage to sort it, but the other way around so WDYT?

**Comment by SylvainTakerkart** [comment]
> So I manage to sort it, but the other way around so WDYT?

the last version you pushed looks perfect ;)

**Comment by SylvainTakerkart** [comment]
Hi @JuliaSprenger @Slowblitz . Can you both proofread before I merge? https://ando.readthedocs.io/en/enh-read_the_doc/ (you probably need to refresh or else the changes don't show up...)

(and yes, @Slowblitz , I finished my sentence ;) )

**Comment by SylvainTakerkart** [comment]
@Slowblitz oooops, no, there's a mistage on the image: the sub-XX_sessions files (tsv and json) should be in the sub-XX directory, not in the ses-YY directory (they describe the different sessions, so they cannot be inside a specific session directory)... can you please update the image?

File: docs/source/index.rst
Comment (JuliaSprenger) [inline docs/source/index.rst:17]:
```suggestion
At this moment, this repository provides several tools, amongst them a validator (:ref:`the AnDOChecker <to_checker>`) and generator functions for user-friendly setup of an AnDO structure. The validator complies with the latest specifications described in the BEP032 document. Please note that this validator is a temporary tool and to be used until BEP032 is officially included in BIDS. From then on the official BIDS validator will replace the AnDOChecker. We are committed to keeping the AnDO tools up to date to follow the latest modifications of the BEP032 that will occur to cope with the feedback from the community.
```

**Review by JuliaSprenger — APPROVED** [review]

**Comment by Slowblitz** [comment]
@SylvainTakerkart like this : 
image

**Comment by SylvainTakerkart** [comment]
> @SylvainTakerkart like this :

yep!

do you have this dataset on disk? just run the AnDOChecker to validate ;) ; and then commit and I'll merge!

**Comment by Slowblitz** [comment]
@SylvainTakerkart so the data set does not validate , so let me correct it and push it after

**Comment by Slowblitz** [comment]
image
WDYT ?

**Comment by SylvainTakerkart** [comment]
> @SylvainTakerkart @JuliaSprenger WDYT ?

what's the difference with the previous one (I don't see it, hehe :dark_sunglasses: )? what was the reason from the non-validation?

**Comment by SylvainTakerkart** [comment]
> > @SylvainTakerkart @JuliaSprenger WDYT ?
> 
> what's the difference with the previous one (I don't see it, hehe dark_sunglasses )? what was the reason from the non-validation?

ps: at this point, I don't think anymore... if you tell me it's validated, it's good to go ;)

**Comment by Slowblitz** [comment]
@SylvainTakerkart so there were no "_ephys" at the end of the data file, and the structured rules did not include the 'task' and 'run' options for data file see #78 I've fixed it I will push it after merging the doc

**Review by SylvainTakerkart — APPROVED** [review]

**Comment by JuliaSprenger** [comment]
Nice, the first version is merged! @Slowblitz So the next step is setting up the readthedocs page and the correct triggers of the doc build.

**Pull request merged**
